gapis/api/templates/specific_gfx_api.cpp.tmpl

```diff
@@ -145,7 +145,13 @@
         {{$args := (GetAnnotation $ "indirect").Arguments}}
         {{$elem := (index $.CallParameters 0).Name}}
         {{$func := Macro "CmdName" $}}
-        if ({{Template "GetIndirectedFunction" "Annotations" $args "Element" $elem "Function" $func}}) {
+        if (
+        #ifdef DEBUG
```

Comment:
The only `#ifdef DEBUG` that I could find in the codebase is actually a `#ifndef NDEBUG`:
https://github.com/google/gapid/blob/2efbe29f4d25dfa279927dc1d84534713884e24c/core/memory_tracker/cc/posix/memory_tracker.inc#L66

I personally prefer `#ifdef DEBUG`, but maybe there's a good reason it hasn't been used in the other case?

Reply:
We use NDEBUG because that is what "assert.h" uses, and is the "standard" way to do things. Not every compiler defined "DEBUG" and NDEBUG I think catches some weird cases that you see in bazel. (What happens when you are building fastbuild vs/ release vs/ debug)

Reply:
I also prefer DEBUG, but changed